rpc/testing.go

```diff
@@ -26,7 +26,7 @@ import (
 
 // NewTestContext returns a rpc.Context for testing.
 func NewTestContext(t *testing.T) *Context {
-	tlsConfig, err := security.LoadTLSConfigFromDir(security.EmbeddedPrefix + "test_certs")
+	tlsConfig, err := security.LoadTLSConfigFromDir("test_certs")
```

Comment:
I'm currently refactoring some of this with much fewer TLS setups. I'll move the test path to an exported const.